spec/ruby_speech/ssml/prosody_spec.rb

```diff
@@ -149,6 +149,24 @@ module SSML
           it "with a negative value" do
             lambda { subject.rate = -100 }.should raise_error(ArgumentError, "You must specify a valid rate ([positive-number](multiplier), :\"x-slow\", :slow, :medium, :fast, :\"x-fast\", :default)")
           end
+
+          it "with a percentage" do
```

Comment:
These should be `context` to enclose the `before` and `its`. An individual example (`it` cannot enclose other examples.